src/test/groovy/dev/morling/jfrunit/JfrSpockSpec.groovy

```diff
@@ -0,0 +1,70 @@
+/**
+ *  Copyright 2020 - 2021 The JfrUnit authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package dev.morling.jfrunit
+
+import org.junit.Rule
+import org.junit.rules.TemporaryFolder
+import spock.lang.Specification
+
+import java.time.Duration
+
+import static dev.morling.jfrunit.ExpectedEvent.event
+import static dev.morling.jfrunit.JfrEventsAssert.assertThat
+
+@JfrEventTest
+class JfrSpockSpec extends Specification {
+
+    @Rule
+    TemporaryFolder temporaryFolder = new TemporaryFolder()
+
+    JfrEvents jfrEvents = new JfrEvents()
+
+    @EnableEvent('jdk.GarbageCollection')
+    @EnableEvent('jdk.ThreadSleep')
+    def 'should Have GC And Sleep Events'() {
+        when:
+        System.gc()
+        sleep(1000)
```

Comment:
For tests, I would try to use shorter sleeps if possible.

Reply:
This one is matching the tests used for JUnit. Should I change even there?

Reply:
I've lowered sleeps across the tests.